shell/common/shell.h

```diff
@@ -243,6 +243,12 @@ class Shell final : public PlatformView::Delegate,
   Rasterizer::Screenshot Screenshot(Rasterizer::ScreenshotType type,
                                     bool base64_encode);
 
+  /// Pauses the calling thread until the first frame is presented.
+  ///\details Don't call this from the GPU thread or the UI thread or you will
```

Comment:
Can you stick to the doxygen commenting scheme followed in the rest of this file. I think we may need a formatting reference for such comments as well. I just picked one that looked good to me but we should gather/document consensus on a format.

Reply:
Done, that style didn't exist when I originally wrote this.